tasks/Module 4/task_7.py

```diff
@@ -0,0 +1,21 @@
+"""Excel-to-CSV Converter."""
+import os
+import csv
+import openpyxl
+
+
+def xl_to_csv(xl_file):
+    """Excel-to-CSV Converter."""
+    os.chdir(os.path.dirname(xl_file) if os.path.dirname(xl_file) else '.')
```

Comment:
Be careful with changing current directory inside your functions. In case of more complex programs with multiple functions it may be frustrating, because you have to remember what functions make which changes.

As an alternative you can work with absolute paths.
```python
input_directory = os.path.dirname(os.path.abspath(xl_file))
# Then inside the for loop:
    output_filename = f'{os.path.splitext(os.path.basename(xl_file))[0]}_{sheet.title}.csv'
    output_filepath = os.path.join(input_directory, output_filename)
    ....
```

Reply:
agree